### ISSUE TITLE
Is there a way we can modify to set a 2 out of 5 threshold for sending?

### ISSUE THREAD

**Issue by nikhil478** [stas-token/stas-sdk#14]
Currently, we can only sign with a single private key. Can we switch to using multisig? @cainnusti ?

**Comment by cainnusti** [comment]
Yes, this is possible, but it will require a new script template. I have spoken to the engineers, and they mentioned that one is in the works but has not been released yet. Once there is a concrete ETA, I will let you know.

The concept includes dynamic multi-signature capabilities that allow custom settings for the number of required signatures.
